File: Tugas coding modul 3/coding python/PRAK305-2310817120005-Noviana_Nur_Aisyah.py
detik = int(input("Input\n"))

hari = detik // (24 * 3600)
jam = (detik % (24 * 3600)) // 3600
menit = (detik % 3600) // 60
detik2 = detik % 60

if hari > 0:
    print(f'\nOutput\n{hari} hari {jam:02d}:{menit:02d}:{detik2:02d}')
else:
    print(f'\nOutput\n{jam:02d}:{menit:02d}:{detik2:02d}')
 
 # 3600 1432 8453 21542 125478

File: Tugas coding modul 3/coding c/PRAK305-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void main(){
    int detik, hari, jam, menit, detik_lagi;

    printf("Input\n");
    scanf("%d", &detik);

    hari = detik / (24 * 3600);
    jam = (detik % (24 * 3600)) / 3600;
    menit = (detik % 3600) / 60;
    detik_lagi = detik % 60;

    if (hari > 0) {
        printf("\nOutput \n%d hari %02d:%02d:%02d", hari, jam, menit, detik_lagi);
    } else{
        printf("\nOutput \n%02d:%02d:%02d", jam, menit, detik_lagi);
    }
}

// 3600  | 1432 | 8453 | 21542 | 125478 